pkg/fuzzer/fuzzer.go

```diff
@@ -27,6 +28,11 @@ type Fuzzer struct {
 	Config *Config
 	Cover  *Cover
 
+	// Number of times coverage for a particular call has overflowed.
+	// Indexed by prog.Syscall.ID + the last element for extra coverage.
+	// Second index: 0 - code coverage, 1 - comparisons.
+	CoverageOverflows [][2]atomic.Uint64
```

Comment:
Nit: at this point we should probably just introduce a struct for it. E.g. `PerCall []SyscallStats` or something along these lines.